src/apps/lwaftr/lwaftr.lua

```diff
@@ -404,7 +390,7 @@ local function drop_ipv4_packet_to_unreachable_host(lwstate, pkt, pkt_src_link)
       lwstate.aftr_mac_inet_side, lwstate.inet_mac, lwstate.aftr_ipv4_ip,
       to_ip, pkt, icmp_config)
 
-   return transmit_icmpv4_reply(lwstate, icmp_dis, pkt, pkt_src_link)
+   return transmit_icmpv4_reply(icmp_dis, pkt, pkt_src_link)
```

Comment:
i would prefer a cleanup here that does

```lua
lwstate:transmit_icmpv4_reply(icmp_dis, pkt, pkt_src_link)
```

Currently we are using global variables for these closures, as if there will only ever be one lwaftr app in a process; that's bogus, no?  WDYT?